src/test/java/seedu/address/testutil/TypicalModules.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.testutil;
+
+import static seedu.address.logic.commands.CommandTestUtil.VALID_CODE_CS3216;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_CREDITS_CS3216;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_NON_CORE;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.module.Module;
+
+/**
+ * A utility class containing a list of {@code Module} objects to be used in tests.
+ */
+public class TypicalModules {
```

Comment:
I only added CS2103T and CS3216 as new modules in this class and left everything else alone first. Likewise, if nobody else needs them, we can remove them at the end